rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/providers/util/PackageNameConverterSpec.groovy

```diff
@@ -122,6 +122,24 @@ class PackageNameConverterSpec extends Specification {
       appVersionStrFromDebPackageName == appVersionStr
   }
 
+  void "if job name contains dashes, they are replaced by hyphen"() {
+    setup:
```

Comment:
Please make the formatting match the neighbors. The `setup`/`when`/`then` clauses have their contents indented, and the constructor args to `BakeRequest` are aligned.